src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -490,10 +490,12 @@ public static partial class Marshal
         public static string PtrToStringUni(System.IntPtr ptr) { return default(string); }
         [System.Security.SecurityCriticalAttribute]
         public static string PtrToStringUni(System.IntPtr ptr, int len) { return default(string); }
+#if NETCOREAPP11
```

Comment:
We have been using the exact casing in our #ifdefs so this would be `netcoreapp11`